src/test/java/com/commercetools/sync/products/helpers/PriceReferenceResolverTest.java

```diff
@@ -141,8 +141,8 @@ public void resolveCustomTypeReference_WithNullIdOnCustomTypeReference_ShouldNot
             .hasFailedWithThrowableThat()
             .isExactlyInstanceOf(ReferenceResolutionException.class)
             .hasMessage("Failed to resolve custom type reference on PriceDraft"
-                + " with country:'DE' and value: 'EUR 10'. Reason: Reference 'id' field"
-                + " value is blank (null/empty).");
+                + " with country:'DE' and value: 'EUR 10'. Reason: The value of 'id' field of the Resource Identifier"
+                + " is blank (null/empty).");
     }
```

Comment:
maybe I would add this "The value of 'id' field of the Resource Identifier is blank (null/empty)" to a variable and reuse this anyhow...

Reply:
agree @ahalberkamp: 03da2ff